closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -1043,12 +1043,9 @@ boolean isRestrictedProviderAllowed(String providerName) {
                 providerName = providerName.substring(0, pos);
```

Comment:
The parameter of this method should be renamed to `providerClassName` to more accurately reflect its meaning.

The removal of the argument does not belong here, but should instead be in `isProviderAllowed(String)` (the only place it might not already be a fully-qualified class name).

Reply:
Yes, and I also renamed to `providerClassName` in method `isRestrictedServiceAllowed(Service service)`.